ChangeLog.md

```diff
@@ -1,3 +1,71 @@
+## 2.8.0 - October 2019
```

Comment:
Normally need to add general section when having new module release
```
### General
* Az.HealthcareApisService 1.0.0 release
```

Reply:
what about the change list of that module?
The change log should be 
### General
* Az.HealthcareApisService 1.0.0 release
  * Updated the powershell version to 1.0.0
  * Updated the SDK version to 1.0.2
  * Update in tests to refer to new SDK version
  * Updated the output structure from nested to flattened.
or just
### General
* Az.HealthcareApisService 1.0.0 release

Reply:
yes